ansible_collections/arista/avd/examples/cv-pathfinder/host_vars/cloudvision/vault.yml

```diff
@@ -0,0 +1,28 @@
+$ANSIBLE_VAULT;1.1;AES256
```

Comment:
I would prefer if this was a regular vars file with inline vault. Makes it easier to understand.

Reply:
yeah so this should never have been there - CV was cleaned up